multiqc/modules/glimpse/err_spl.py

```diff
@@ -0,0 +1,305 @@
+                "min": 0,
+                "scale": "YlRd",
+            },
+            "AA_hom_mismatches": {
+                "title": "Alternate-Alternate homozygous mismatches",
+                "description": "Number of Alternate-Alternate hom mismatches",
+                "min": 0,
+                "scale": "YlRd",
+            },
+            "RR_hom_mismatches_rate_percent": {
+                "title": "Reference-Reference homozygous mismatches rate",
+                "description": "Rate of Reference-Reference hom mismatches",
+                "min": 0,
+                "max": 100,
+                "suffix": "%",
+                "scale": "YlRd",
+            },
+            "RA_het_mismatches_rate_percent": {
+                "title": "Reference-Alternate heterozygous mismatches rate",
+                "description": "Rate of Reference-Alternate het mismatches",
+                "min": 0,
+                "max": 100,
+                "suffix": "%",
+                "scale": "YlRd",
+            },
+            "AA_hom_mimatches": {
+                "title": "Alternate-Alternate homozygous mismatches rate",
+                "description": "Rate of Alternate-Alternate hom mismatches",
+                "min": 0,
+                "max": 100,
+                "suffix": "%",
+                "scale": "YlRd",
+            },
+            "non_reference_discordanc_rate_percent": {
+                "title": "Non-reference discordance rate",
+                "description": "Rate of non-reference discordance",
+                "min": 0,
+                "max": 100,
+                "suffix": "%",
+                "scale": "YlRd",
+            },
+            "best_gt_rsquared": {
+                "title": "Best genotype r-squared",
+                "description": "Best genotype r-squared",
+                "min": 0,
+                "max": 1,
+                "scale": "YlGn",
+            },
+            "imputed_ds_rsquared": {
+                "title": "Imputed dosage r-squared",
+                "description": "Imputed dosage r-squared",
+                "min": 0,
+                "max": 1,
+                "scale": "YlGn",
+            },
+        }
+        self.add_section(
+            name="Genotype concordance by samples",
+            anchor="glimpse-err-spl-table-section",
+            description=(
+                "Stats parsed from <code>GLIMPSE2_concordance</code> output, and summarized across all samples."
+            ),
+            plot=table.plot(
+                data_by_sample,
+                deepcopy(headers),
+                pconfig={
+                    "id": "glimpse-err-spl-table",
+                    "title": "Glimpse concordance: errors by sample summary",
+                },
+            ),
+        )
+
+        # TODO: Question for @LouisLeNezet - what is table_data?
+        # self.general_stats_addcols(table_data, headers, namespace="err_spl")
```

Comment:
@LouisLeNezet - this was throwing an error as `table_data` is not defined anywhere. I commented it out here, but if it's a relic of moving the code from somewhere else and you don't want any data in general stats, we should delete it:

```suggestion
```